doc/user_guide/docs/visualization.md

```diff
@@ -18,6 +18,8 @@ One way to enable visualization is to export a visualization file every time ste
 (or every N time steps). In the `bmd.toml` file add the following lines:
 
 ```
+[simulation]
+/// For other simulations certain parameters would also need to be defined within this section of the toml file.
```

Comment:
Comments in a toml file start with `#`.
https://github.com/toml-lang/toml#comment

Reply:
Corrected the comment style.